packages/crypto/src/ecvrf.rs

```diff
@@ -139,6 +139,10 @@ fn ecvrf_decode_proof(pi: &[u8]) -> CryptoResult<((Mpz, Mpz), Mpz, Mpz)> {
     let c = parse_rev_bytes(&pi[32..48]);
     let s = parse_rev_bytes(&pi[48..]);
 
+    if s >= *ORDER {
+        return Err(CryptoError::generic_err("s is out of range"));
```

Comment:
Should we add test cases for errors as well?

Reply:
Agree